cost-analyzer/templates/cost-analyzer-deployment-template.yaml

```diff
@@ -18,11 +18,11 @@ spec:
   selector:
     matchLabels:
 {{ include "cost-analyzer.selectorLabels" . | nindent 8}}
-  strategy:
-    rollingUpdate:
-      maxSurge: 1
-      maxUnavailable: 1
-    type: RollingUpdate
+{{- if .Values.kubecostDeployment }}
+{{- with .Values.kubecostDeployment.deploymentStrategy }}
+  strategy: {{ toYaml . | nindent 4 }}
+{{- end }}
+{{- end }}
```

Comment:
Can we do a default RollingUpdate in an else block just in case users haven't updated their values.yaml?

Reply:
Even if they don't add it, the default is in the new chart. https://github.com/kubecost/cost-analyzer-helm-chart/pull/1746/files#diff-e232ea8a3d165b37f6212249d3718bb96e05b64a591f1fa36d77c9448c747527R671

Tested myself with no error.

No?

Reply:
Oh I guess that's a fair point.

Reply:
So I changed that, and I kind of like it better. Adding the default option, it's more in line with how it was previously with no value in the values.yaml, and I can now set defaults for

```
    rollingUpdate:
      maxSurge: 1
      maxUnavailable: 1
```